src/include/btmem.h

```diff
@@ -248,17 +248,14 @@ struct __wt_page_lookaside {
     wt_timestamp_t max_ondisk_ts;  /* Maximum timestamp on disk */
     wt_timestamp_t min_skipped_ts; /* Skipped in favor of disk version */
     bool has_prepares;             /* One or more updates are prepared */
-    WT_ITEM max_las_key;           /* The maximum key in the LAS for the page */
-    WT_ITEM min_las_key;           /* The minimum key in the LAS for the page */
     struct __wt_birthmark_details {
```

Comment:
I'd like to rename this from `WT_BIRTHMARK_DETAILS` to `WT_KEY_MEMENTO` since this only indicates a birthmark when `txnid == WT_TXN_ABORTED`. What do people think of this?